GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivityBase.java

```diff
@@ -123,6 +121,7 @@ public void run() {
     abstract GVRActivityNative makeActivityNative();
     abstract GVRViewManager makeViewManager(GVRXMLParser xmlParser);
     abstract GVRMonoscopicViewManager makeMonoscopicViewManager(GVRXMLParser xmlParser);
+    abstract GoogleVRViewManager makeGoogleVRViewManager(GVRXMLParser xmlParser);
```

Comment:
No backend-specifics in the common files. This could be all handled in the oculus backed.

1. revert the changes to onInitAppSettings (226-247)
2. revert changes on line 76 (move to GVRActivity.java) and line 124
3. change GVRActivity.java along these lines
```
    @Override
    protected final GVRViewManager makeViewManager(final GVRXMLParser xmlParser) {
        if (!mUseFallback) {
            return new GVRViewManager(this, getScript(), xmlParser);
        } else {
            return new GoogleVRViewManager(this, getScript(), xmlParser);
        }
    }
```

PS What is the problem that requires the change on 247?